src/main/java/com/google/maps/model/WifiAccessPoint.java

```diff
@@ -0,0 +1,39 @@
+package com.google.maps.model;
+
+/**
+ * WiFi access point objects
+ *
+ * The request body's wifiAccessPoints array must contain two or more WiFi access point objects.
+ * {@code macAddress} is required; all other fields are optional.
+ *
+ * <p>Please see <a href="https://developers.google.com/maps/documentation/geolocation/intro#wifi_access_point_object">
+ * WiFi Access Point Objects</a> for more detail.
+ *
+ */
+public class WifiAccessPoint {
+  /**
+  * {@code macAddress}: (required) The MAC address of the WiFi node. Separators must be : (colon) and
+  * hex digits must use uppercase.
+  */
+  public String macAddress;
```

Comment:
When you move across to the builder style, can you ensure you add a validator for this field?